frontend/src/lib/k8s/api/v2/hooks.ts

```diff
@@ -128,7 +141,7 @@ export function useKubeObject<K extends KubeObject>({
     enabled: !!endpoint && !!data,
     cluster,
     onMessage(update) {
-      if (update.type !== 'ADDED' && update.object) {
+      if (isKubeListUpdateEvent(update) && update.type !== 'ADDED' && update.object) {
```

Comment:
please move this logic inside useWebSocket, onMessage should only receive update events like before